### DISCUSSION TITLE
NFT Token Id

### DISCUSSION THREAD

**Discussion by Samir-Alam** [Cyfrin/foundry-full-course-cu#723] [Q&A]
how do I get the token Id of the NFT that I deployed on the sepolia testnet

**Comment by theirrationalone** [comment]
Mint your nft first... Then, 
Make a helper function to get current tokenId Or tokenCounter. As you know we incremented our tokenCounter Or tokenId during minting, that's why we would need to decrement value by 1 after getting it from helper function.

**Comment by Niferu** [comment]
Hi @Samir-Alam 

It is also good to understand that you are deploying contract that generates(mints) NFT's, you are not deploying NFT. In your contract that creates NFT's you can make `getTokenIds()` function, which will return table of all tokenId's created.
